kiwiapp-public/src/components/home/HomeBanner.tsx

```diff
@@ -0,0 +1,59 @@
+"use client";
+import Image from "next/image";
+import OverviewPreview from "public/assets/images/overview-preview.png";
```

Comment:
`overview-preview.png` seems to be badly generated. e.g. the text here isn't clear at all.
Can you make sure you export PNGs from Figma directly? 
![image](https://github.com/user-attachments/assets/1714639a-dc8d-4329-9f3c-388017def8dd)